src/block-management/block-toolbar.js

```diff
@@ -1,11 +1,28 @@
+/**
+ * @format
+ * @flow
+ */
+
 import React, { Component } from 'react';
 import { View } from 'react-native';
+import { IconButton } from '@wordpress/components';
 import { BlockFormatControls, BlockControls } from '@wordpress/editor';
+import { __ } from '@wordpress/i18n';
+
+type PropsType = {
+	onInsertClick: void => void,
+};
 
-export default class BlockToolbar extends Component {
+export default class BlockToolbar extends Component<PropsType> {
 	render() {
 		return (
 			<View style={ { height: 50, backgroundColor: '#DCDCDC', flexDirection: 'row' } }>
+				<IconButton
+					className="block-list-appender__toggle"
+					label={ __( 'Add block' ) }
+					icon="insert"
```

Comment:
The only concern here would be that we didn't share some constant with gutenberg project, so in the future, if someone changes those strings, it will break gutenberg-mobile.

Reply:
Sounds like a good plan :)